src/test/java/com/faforever/client/chat/ChatUserItemControllerTest.java

```diff
@@ -98,42 +87,6 @@ public void testGetPlayer() {
     assertThat(instance.getChatUser(), is(chatUser));
   }
 
-  @Test
-  public void testNullValuesHidesNodes() {
-    PlayerBean player = PlayerBeanBuilder.create().defaultValues().get();
-    ChatChannelUser chatUser = ChatChannelUserBuilder.create(player.getUsername(), CHANNEL_NAME)
-        .defaultValues()
-        .player(player)
-        .get();
-    instance.setChatUser(chatUser);
-    WaitForAsyncUtils.waitForFxEvents();
-
-    assertFalse(instance.avatarImageView.isVisible());
-    assertFalse(instance.playerStatusIndicator.isVisible());
-    assertFalse(instance.playerMapImage.isVisible());
-    assertFalse(instance.countryImageView.isVisible());
-  }
-
-  @Test
-  public void testNotNullValuesShowsNodes() {
-    PlayerBean player = PlayerBeanBuilder.create().defaultValues().get();
-    ChatChannelUser chatUser = ChatChannelUserBuilder.create(player.getUsername(), CHANNEL_NAME)
-        .defaultValues()
-        .player(player)
-        .avatar(new Image(UiService.UNKNOWN_MAP_IMAGE))
-        .countryFlag(new Image(UiService.UNKNOWN_MAP_IMAGE))
-        .mapImage(new Image(UiService.UNKNOWN_MAP_IMAGE))
-        .statusImage(new Image(UiService.UNKNOWN_MAP_IMAGE))
-        .get();
-    instance.setChatUser(chatUser);
-    WaitForAsyncUtils.waitForFxEvents();
-
-    assertTrue(instance.avatarImageView.isVisible());
-    assertTrue(instance.playerStatusIndicator.isVisible());
-    assertTrue(instance.playerMapImage.isVisible());
-    assertTrue(instance.countryImageView.isVisible());
-  }
```

Comment:
Why were these just removed instead of replaced or made new?

Reply:
I am not sure what you mean about making sure the code has been reviewed. And I think the tests are important especially for the chat user list as many things broke without being noticed before in the past

Reply:
I know tests are important and will write the tests. I mean that I have to rewrite the tests if the methods can be changed or updated during the review.

Reply:
You don't like the implementation of the `example' method, I'm correcting it and I still have to correct or rewrite the `example` test method

Reply:
Ah okay gotcha